core/controllers/topic_viewer.py

```diff
@@ -73,9 +73,15 @@ def get(self, topic_name):
             summary.to_human_readable_dict() for summary
             in additional_story_summaries]
 
+        skill_ids = topic.get_all_skill_ids()
+
+        subtopics = topic.get_all_subtopics()
+
         self.values.update({
             'topic_name': topic.name,
             'canonical_story_dicts': canonical_story_dicts,
-            'additional_story_dicts': additional_story_dicts
+            'additional_story_dicts': additional_story_dicts,
+            'skill_ids': skill_ids,
```

Comment:
So, this field would be `uncategorized_skill_ids`.

Reply:
Done